module1.test.js

```diff
@@ -128,11 +128,19 @@ describe("Module 1 Answers", function() {
     ]);
   });
 
-  xit("Q8 - Tax addition", function() {
+  it("Q8 - Tax addition", function() {
     // tax is 20% of the retail price
     const taxAmount = 0.2;
     const retailPrice = "£10";
     solution = exercises.question8(retailPrice, taxAmount);
     expect(solution).toEqual("£12");
   });
+
+  it("Q9 - Tax addition", function() {
```

Comment:
Did you add the function `question9 in order to test it? You're still testing `question8` in here though?
I think you just wanted to test a different input, is that right? Look a the example of testing Question 5 - we're testing the same function there with different inputs - you can try the same with question 8 test :)